src/functionalTest/java/uk/gov/hmcts/reform/iacasenotificationsapi/util/MapValueExtractor.java

```diff
@@ -27,15 +36,94 @@ private MapValueExtractor() {
                 return null;
             }
 
+            Map map1 = (Map) value;
+
+            if (featureFlag == false) {
+
+                if (map1.containsKey("body")) {
```

Comment:
@burale lets change this from `map1` to `expectationMap`

Reply:
Done